Source/ASNetworkImageNode.mm

```diff
@@ -206,7 +208,9 @@ - (void)setURL:(NSURL *)URL resetToDefault:(BOOL)reset
     _imageWasSetExternally = NO;
     
     [self _locked_cancelImageDownloadWithResumePossibility:NO];
-
+    
+    [self _locked_setLoadingProgress:0.0];
```

Comment:
Should we do something similar to line 221?

Reply:
I suggest set `0.0` directly, because we actually don't do download operation?